Apps/WebClient/src/ClientApp/src/models/configData.ts

```diff
@@ -79,3 +84,15 @@ export interface TimeOutsConfiguration {
     // Gets or sets the aount of time in seconds that a user will have to wait between resending an SMS verification code.
     resendSMS: number;
 }
+
+// Configuration for offline mode.
+export interface OfflineModeConfiguration {
+    // The start datetime for offline mode.
+    StartDateTime: StringISODate;
+    // The end datetime for offline mode.
+    EndDateTime: StringISODate;
+    //The message to display if in offline mode
+    Message: StringISODate;
+    //The list of IPs that can connect during offline mode.
+    IPWhitelist: Array<string>;
```

Comment:
Can we rename Whitelist to IPWhitelist everywhere (including the json in the config file) to be consistent?

Reply:
I say we make Whitelist everywhere.

Reply:
I think that Whitelist in the object makes sense I'll change it and ensure we are consistent everywhere.